lib/shopify_api/resources/application_charge.rb

```diff
@@ -2,6 +2,12 @@ module ShopifyAPI
   class ApplicationCharge < Base
     undef_method :test
 
+    class << self
+      [:pending, :cancelled, :accepted, :declined].each do |status|
+        define_method(status) { (all || []).select { |c| c.status == status.to_s } }
```

Comment:
This type of problem has burned us before. But I agree the API should not change the case on important strings like these.